src/main/kotlin/app/revanced/patches/youtube/layout/watermark/patch/HideWatermarkPatch.kt

```diff
@@ -35,11 +35,11 @@ class HideWatermarkPatch : BytecodePatch(
     override fun execute(context: BytecodeContext): PatchResult {
         SettingsPatch.PreferenceScreen.LAYOUT.addPreferences(
             SwitchPreference(
-                "revanced_branding_watermark_enabled",
-                StringResource("revanced_branding_watermark_enabled_title", "Show branding watermark"),
-                false,
-                StringResource("revanced_branding_watermark_summary_on", "Branding watermark is shown"),
-                StringResource("revanced_branding_watermark_summary_off", "Branding watermark is hidden")
+                "revanced_branding_video_watermark_hidden",
+                StringResource("revanced_branding_video_watermark_hidden_title", "Hide video branding watermark"),
```

Comment:
It is not a video branding, but a channel branding. To address consistency, please remove "Video playback" here.

Reply:
I changed it to "Hide branding video watermark"